scripts/signedbuildskadi.sh

```diff
@@ -24,6 +24,7 @@ if [ $default_skadi_passwords = "false" ]
     TIMESKETCH_PASSWORD=$(openssl rand -base64 32 |sha256sum | sed 's/ //g')
     NGINX_USER="skadi_$(openssl rand -base64 3)"
     NGINX_PASSWORD=$(openssl rand -base64 32 |sha256sum | sed 's/ //g')
+    HEAP_SIZE=$(HEAP_SIZE:-1g)
```

Comment:
After testing the script, I don't think these actually need to be here. Let's try removing both of these and then we can merge and test

Reply:
latest commit fixes this